Use the version of orcc in the sysroot not the one from the host

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>
Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>

Upstream-Status: Inappropriate [configuration]

Index: schroedinger-1.0.9/configure.ac
===================================================================
--- schroedinger-1.0.9.orig/configure.ac
+++ schroedinger-1.0.9/configure.ac
@@ -54,7 +54,7 @@ if test "x${HAVE_ORC}" != xyes ; then
   AC_ERROR([orc-0.4 >= $ORC_VER is required])
 fi
 SCHRO_PKG_DEPS="$SCHRO_PKG_DEPS orc-0.4 >= $ORC_VER"
-ORCC=`$PKG_CONFIG --variable=orcc orc-0.4`
+ORCC=$STAGING_DIR`$PKG_CONFIG --variable=orcc orc-0.4`
 AC_SUBST(ORCC)
 
 AC_ARG_WITH([thread],
